scripts/build-fixtures.js

```diff
@@ -15,6 +15,7 @@ const fixtures = [
     ['moduleSourceName', {
         moduleSourceName: 'react-i18n',
     }],
+    'descriptionsAsObjects'
```

Comment:
Can you move this right after `defineMessages` so the fixtures remain in alpha order?